src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -2582,9 +2574,9 @@ def test_resource_policyset_subscription_id(self, resource_group):
         if not self.in_recording:
             with mock.patch('azure.cli.command_modules.resource.custom._get_subscription_id_from_subscription',
                             return_value=MOCKED_SUBSCRIPTION_ID):
-                self.resource_policyset_operations(resource_group, None, '0b1f6471-1bf0-4dda-aec3-cb9272f09590')
+                self.resource_policyset_operations(resource_group, None, 'f67cc918-f64f-4c3f-aa24-a855465f9d41')
         else:
-            self.resource_policyset_operations(resource_group, None, '0b1f6471-1bf0-4dda-aec3-cb9272f09590')
+            self.resource_policyset_operations(resource_group, None, 'f67cc918-f64f-4c3f-aa24-a855465f9d41')
```

Comment:
Please do not replace the tenant id used by the CLI team. If you don't have permission for this tenant, I can help you re-record yaml file

Reply:
updated. It seems the mock.patch helped the test pass.